packages/Testing/Fixture/FixtureSplitter.php

```diff
@@ -26,15 +26,15 @@ public static function split(string $filePath): array
     }
 
     /**
-     * @return array<int, string>
+     * @return array{string, string}
      */
     public static function splitFixtureFileContents(string $fixtureFileContents): array
     {
-        $posixContents = explode("-----\n", $fixtureFileContents);
+        $posixContents = explode("-----\n", $fixtureFileContents, 2);
```

Comment:
it can be 3 and more, see `TagValueNodeReprintTest`

Reply:
Interessting.. never worked with these tests